generator/AWSPSGeneratorLib/HelpMaterials/Examples/APS/Unregister-APSUserStackBatch.xml

```diff
@@ -0,0 +1,8 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!--Examples for the AWS PowerShell CmdLets-->
+<examples>
+  <example>
+    <code> Unregister-APSUserStackBatch -UserStackAssociation (New-Object -TypeName Amazon.AppStream.Model.UserStackAssociation -Property @{AuthenticationType="USERPOOL";SendEmailNotification=$False;StackName="PowershellStack";UserName="TestUser1@lab.com"})</code>
```

Comment:
It can be simplified to `-UserStackAssociation @{AuthenticationType="USERPOOL";SendEmailNotification=$False;StackName="PowershellStack";UserName="TestUser1@lab.com"}`